.github/workflows/code-formating-check.yml

```diff
@@ -0,0 +1,29 @@
+name: "Check Formating"
+on: [push, pull_request]
+
+jobs:
+  check_code_formatting:
+    name: Check code formatting with Black
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v1.1.1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Restore cache for check_code_formatting
+      uses: actions/cache@v1
+      with:
+        path: ~/.cache/pip
+        key: check_code_formatting-${{ matrix.python-version }}
+        restore-keys: check_code_formatting-${{ matrix.python-version }}
+    - name: Update pip
+      run: python -m pip install --upgrade pip
+    - name: Install Black
+      run: pip install black==19.3b0
```

Comment:
not sure about beta here

Reply:
It is pretty old one, from 14 march 2019: https://github.com/psf/black/tree/19.3b0
Works good

Reply:
Seems like all his versions are either tagged as alpha or beta. I guess this is, since he does not think of his framework to be "production ready" yet. Same what we indicate by releases < 1.0